### DISCUSSION TITLE
404 on http://kimai/en/homepage

### DISCUSSION THREAD

**Discussion by iandoug** [kimai/kimai#3164] [On-Premise: Installation & Updates]
Hi 

Running on localhost, nginx.

Loading the site redirects to http://kimai/en/homepage with a 404.

Before I go searching for the wrong problem, is Kimai supposed to redirect to such a page?

Thanks, Ian

**Comment by kevinpapst** [comment]
yes: https://www.kimai.org/documentation/webserver-configuration.html

**Reply by iandoug** [reply]
Hi Kevin

Thanks for fast answer.
I don't have /en/ under public ...  only these folders.
avatars
build
bundles
favicon

Did some step in the build process break?

Thanks, Ian

**Comment by iandoug** [comment]
i redid this from scratch and there is still no public/en/  folder. Is this supposed to be created during install because I don't see it in the code repo either.

Thanks, Ian

**Reply by kevinpapst** [reply]
This is a virtual URL. You are overthinking this. It is supposed to work if you follow the instructions.

**Reply by iandoug** [reply]
I did, as far as I could see ... but still get 404 on http://kimai/en/homepage after typing in http://kimai 

Where can I look for issue?

127.0.0.1 - - [22/Feb/2022:22:11:17 +0200] "GET / HTTP/1.1" 302 306 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:91.0) Gecko/20100101 Firefox/91.0"
127.0.0.1 - - [22/Feb/2022:22:11:17 +0200] "GET /en/homepage HTTP/1.1" 404 153 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:91.0) Gecko/20100101 Firefox/91.0"
127.0.0.1 - - [22/Feb/2022:22:11:17 +0200] "GET /favicon.ico HTTP/1.1" 200 809 "http://kimai/en/homepage" "Mozilla/5.0 (X11; Linux x86_64; rv:91.0) Gecko/20100101 Firefox/91.0"
127.0.0.1 - - [22/Feb/2022:23:14:33 +0200] "GET / HTTP/1.1" 302 306 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:91.0) Gecko/20100101 Firefox/91.0"
127.0.0.1 - - [22/Feb/2022:23:14:33 +0200] "GET /en/homepage HTTP/1.1" 404 153 "-" "Mozilla/5.0 (X11; Linux x86_64; rv:91.0) Gecko/20100101 Firefox/91.0"


2022/02/22 23:14:33 [error] 26427#26427: *1357 open() "/home/ian/1web/kimai2/public/en/homepage" failed (2: No such file or directory), client: 127.0.0.1, server: kimai, request: "GET /en/homepage HTTP/1.1", host: "kimai"

Thanks, Ian

**Comment by Brigithubert** [comment]
Same issue here. Did you find any solution?

**Reply by iandoug** [reply]
No, http://old.myhours.com put their old "basic" site back up (I guess enough people complained) so I went back to using it.
